lib/common/schemas.c

```diff
@@ -489,26 +489,33 @@ pcmk__sort_schemas(void)
 void
 crm_schema_init(void)
 {
-    const char *remote_schema_dir = pcmk__remote_schema_dir();
-    char *base = pcmk__xml_artefact_root(pcmk__xml_artefact_ns_legacy_rng);
-    const pcmk__schema_version_t zero = SCHEMA_ZERO;
+    static bool needs_init = true;
 
-    wrap_libxslt(false);
+    if (needs_init) {
```

Comment:
Actually we rely on calling `crm_schema_init()` multiple times for the new remote schema stuff (see the call in `remoted_request_cib_schema_files()`). This will need to be a little more aware.

Reply:
Hmm, okay I'll look into that. I had one or more unit tests fail until I added this guard.

Reply:
I'm guessing it should be sufficient to add a force argument. If initialization has already been done, without force it would be a no-op, with force it would do a schema_cleanup and re-initialization. I haven't traced all callers to verify that though.

Reply:
Either that or have `crm_schema_cleanup()` set the `needs_init` flag back to false